cmake/test.cmake

```
@@ -0,0 +1,21 @@
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
+)
+# For Windows: Prevent overriding the parent project's compiler/linker settings
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+
+
+enable_testing()
+FILE(GLOB cpp_tests pyg_lib/test/cpp/test_*.cpp)
```

```suggestion
FILE(GLOB tests test/csrc/**/test_*.cpp)
```
Let's move tests to the root dir.